static/js/recent_topics_util.js

```diff
@@ -3,10 +3,7 @@ import $ from "jquery";
 import * as compose_state from "./compose_state";
 import * as overlays from "./overlays";
 import * as popovers from "./popovers";
-
-export function is_visible() {
-    return $("#recent_topics_view").is(":visible");
-}
+import {is_visible} from "./recent_topics_ui";
```

Comment:
@showell may remember, but I think moving this function may have consequences for the import graph that tend to increase the presence of import cycles.  If that's indeed an issue, we could address that by adding a `set_is_visible` method here that's called from `recent_topics_ui.js`.